packages/core-design-system/scripts/constants.js

```diff
@@ -0,0 +1,45 @@
+import { formats } from 'style-dictionary/enums'
+
+export const DESIGN_SYSTEM_ROOT = 'src/styles'
+export const DESIGN_SYSTEM_ROOT_ESM = 'src/styles-esm'
+
+export const DESIGN_SYSTEM_PREFIX = 'cn'
+
+export const STYLE_BUILD_FORMATS = {
+  CSS_VARIABLES: formats.cssVariables,
+  JS_ESM: formats.javascriptEsm
+}
+
+/**
+ * Shared common config for css and ts style generation
+ */
+export const COMMON_CONFIG = {
+  transformGroup: 'tokens-studio',
+  prefix: DESIGN_SYSTEM_PREFIX,
+  options: {
+    fileHeader: () => {
+      return [
+        'Harness Design System',
+        'Generated style tokens - DO NOT EDIT DIRECTLY',
+        `Generated on ${new Date().toUTCString()}`,
```

Comment:
Again, this will be overwritten every time the build runs, regardless of whether the content changes.